packages/world/src/modules/core/CoreRegistrationSystem.sol

```diff
@@ -3,23 +3,18 @@ pragma solidity >=0.8.21;
 
 import { IWorldErrors } from "../../IWorldErrors.sol";
 
-import { AccessManagementSystem } from "./implementations/AccessManagementSystem.sol";
-import { BalanceTransferSystem } from "./implementations/BalanceTransferSystem.sol";
-import { BatchCallSystem } from "./implementations/BatchCallSystem.sol";
 import { ModuleInstallationSystem } from "./implementations/ModuleInstallationSystem.sol";
 import { StoreRegistrationSystem } from "./implementations/StoreRegistrationSystem.sol";
 import { WorldRegistrationSystem } from "./implementations/WorldRegistrationSystem.sol";
 
 /**
- * @title Core System for World
- * @notice This system aggregates all World functionalities externalized from the World contract, aiming to keep the World contract's bytecode lean.
+ * @title Core Registration System for World
+ * @notice This system aggregates World registration and installation functionalities externalized from the World contract, aiming to keep the World contract's bytecode lean.
  * @dev Aggregates multiple system implementations for the World.
+ * Split into multiple systems because of the bytecode size limit.
```

Comment:
```suggestion
```
This comment makes sense in the context of this PR but not when reading this by itself file afterwards